drafts/WEP0002-slack-community-moderation-procedures.rst

```diff
@@ -0,0 +1,142 @@
+* **Recruiter spam** - while it's not inherently bad for recruiters to find candidates within the Slack community, we want to make sure they don't detract from it in doing so. Spamming the community with job postings (and thereby distracting from posts from users who are affiliated with the postings) and pushy DMs give the impression that they're trying to mine the community for financial reasons, and that's not what we want. Because moderators can't monitor DMs, we depend on users to speak up when they happen in order to take action, and should communicate clearly that they can and should.
+* **Marketing posts in #intros** - the purpose of the #intros channel is to get to know new members of the community. It's often a difficult line to draw between a person and their work, but a general rule of thumb is that users shouldn't be looking for sales prospects in that channel. Posts including things like personal websites and social media links with invitations to connect are okay; however, writing on behalf of a business looking for potential clients would fall under sales-y content and thus would be subject to removal. At that point, a moderator can send the user a message explaining the removal, along with a warm invitation to post again without the sales angle.
+* **Job-seeking advertisements** - at the time of writing, we are in the thick of the COVID-19 pandemic, which has made it a particularly difficult time to look for work. While we definitely wouldn't want to discourage users from using the Slack community to find work, we also want to make sure they're doing so in the proper channels - #work-wanted and #career-advice are a good place to start, rather than the posts that sometimes pop up in channels like #general.
+
+With some of these rules, we've been more lenient towards long-time contributors to the community with whom we've built trust over time. In those cases, we can trust that the community will recognize them and know they're not looking for business prospects. If you're unsure, searching through the user's past posts or checking out their profile can offer clues about other ways in which they've engaged with the community and whether they're affiliated with a product. Some detective work can be useful in identifying whether a complimentary post is sales-y or an honest endorsement.
+
+With these better defined post guidelines in place, members of the Slack community can feel confident that they can interact with each other as individuals and as professionals, rather than as business prospects, and moderators can more easily remove roadblocks to that end.
+
+Code of conduct violations
```

Comment:
I feel like this might be better under a heading of "Three strikes rule" or similar, as this is a bit broader than explicit COC violations.

Reply:
It doesn't add much, so I opted to just take it out instead of letting it get in the way. :)